go.mod

```diff
@@ -3,26 +3,28 @@ module github.com/k8snetworkplumbingwg/sriov-cni
 go 1.19
 
 require (
-	github.com/containernetworking/cni v1.1.2
-	github.com/containernetworking/plugins v1.1.1
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.17.0
-	github.com/stretchr/testify v1.5.1
-	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
-	golang.org/x/net v0.7.0
+	github.com/containernetworking/cni v1.1.3-0.20230206011450-121798a08dcc
```

Comment:
nit: Is bumping `github.com/containernetworking/cni` strictly necessary?
I'd prefer to not setting this dep to a non released one, if possible.

Reply:
Unfortunately yes, as the current release of  github.com/containernetworking/cni (1.1.2) pulls in an older version of ginkgo, which by extension pulls in the outdated package.

i.e. cni@v1.1.2 -> ginkgo@v2.1.3->slim-sprig->testify@v1.5.1->yaml.v2@v2.2.2